subscription_oca/__manifest__.py

```diff
@@ -3,15 +3,16 @@
 {
     "name": "Subscription management",
     "summary": "Generate recurring invoices.",
-    "version": "17.0.1.0.0",
+    "version": "18.0.1.0.0",
     "development_status": "Beta",
     "category": "Subscription Management",
     "website": "https://github.com/OCA/contract",
     "license": "AGPL-3",
-    "author": "Domatix, Odoo Community Association (OCA)",
+    "author": "Domatix, Onestein, Odoo Community Association (OCA)",
```

Comment:
@tarteo Are you really author? Migration does not mean necessarily you become one.